### ISSUE TITLE
addEventListener cannot bind events directly to window or svg elements in 3.0.6

### ISSUE THREAD

**Issue by pgrippi** [ember-lifeline/ember-lifeline#162]
The [findElement](https://github.com/ember-lifeline/ember-lifeline/blob/master/addon/mixins/dom.ts#L141) method was updated in 3.0.6 to use this check to see if an element was passed into `addEventListener`:

```
selector instanceof HTMLElement && (selector.nodeType || selector instanceof Window)
```

Both window and svg elements fail to pass the first instanceof check

**Comment by rwjblue** [comment]
Code from prior version:

https://github.com/ember-lifeline/ember-lifeline/blob/89a1bb823874ddd5f626cb05a66cd8600ef6d5e9/addon/mixins/dom.ts#L140-L147

**Comment by rwjblue** [comment]
Once we fix, we should add tests for both SVG and window to ensure we do not regress again...

**Comment by scalvert** [comment]
Resolved in v3.0.7 🎉
